### DISCUSSION TITLE
latest nightly build but very old sqlite version

### DISCUSSION THREAD

**Discussion by qrockz** [sqlitebrowser/sqlitebrowser#3494] [Q&A]
Hi,

I'm confused. I've downloaded the latest nightly build for Linux as AppImage;
But when I want to use window functionality like `ROW_NUMBER() OVER (....` I'll get errors.

`DB Browser for SQLite Version 3.12.99 (Oct 31 2023)`
So why seems this AppImage using **sqlite 3.12.99** and not the latest version of sqlite (or some new Version)?

**Comment by lucydodo** [comment]
Hi, @qrockz 

The **3.12.99** you mention indicates that it is a nightly release for the 3.12 release family.
When I check our CI logs, we used SQLite version 3.43.2 for the last AppImage build,
and in DB4S you can check the dynamically embedded SQLite version via the run `select sqlite_version();`

I'd love to try it, but I don;t have an x86_64 machine available at the moment. :(

**Comment by justinclift** [comment]
@qrockz In the version string:

```
DB Browser for SQLite Version 3.12.99 (Oct 31 2023)
````

The "3.12.99" there is the version of our application, not the version of SQLite. :wink:

That being said, it's *supposed* to be including a pretty recent version of SQLite.

If you run this SQL query in the "Execute SQL" tab, it should tell you the exact version of SQLite it's actually using:

```
SELECT sqlite_version();
```

**Reply by mgrojo** [reply]
From
https://github.com/sqlitebrowser/sqlitebrowser/releases/download/continuous/DB_Browser_for_SQLite-dev-96b1b6b-x86_64.AppImage
I get SQLite Version 3.43.2, both from the About and from the SQL function.
